README.md

```diff
@@ -1,78 +1,62 @@
+    <img src="logo_outlined.png" width="400" alt="Redot Engine logo">
   </a>
 </p>
 
 ## 2D and 3D cross-platform game engine
 
-**[Godot Engine](https://godotengine.org) is a feature-packed, cross-platform
+**[Redot Engine](https://redotengine.org) is a feature-packed, cross-platform
 game engine to create 2D and 3D games from a unified interface.** It provides a
-comprehensive set of [common tools](https://godotengine.org/features), so that
+comprehensive set of common tools, so that
 users can focus on making games without having to reinvent the wheel. Games can
 be exported with one click to a number of platforms, including the major desktop
 platforms (Linux, macOS, Windows), mobile platforms (Android, iOS), as well as
-Web-based platforms and [consoles](https://docs.godotengine.org/en/latest/tutorials/platform/consoles.html).
+Web-based platforms and consoles.
 
 ## Free, open source and community-driven
 
-Godot is completely free and open source under the very permissive [MIT license](https://godotengine.org/license).
+Redot is a completely free and open source fork of Godot under the very permissive MIT license.
 No strings attached, no royalties, nothing. The users' games are theirs, down
-to the last line of engine code. Godot's development is fully independent and
+to the last line of engine code. Redot's development is fully independent and truly
 community-driven, empowering users to help shape their engine to match their
-expectations. It is supported by the [Godot Foundation](https://godot.foundation/)
-not-for-profit.
+expectations.
 
 Before being open sourced in [February 2014](https://github.com/godotengine/godot/commit/0b806ee0fc9097fa7bda7ac0109191c9c5e0a1ac),
 Godot had been developed by [Juan Linietsky](https://github.com/reduz) and
-[Ariel Manzur](https://github.com/punto-) (both still maintaining the project)
+[Ariel Manzur](https://github.com/punto-) (both still maintaining Godot)
 for several years as an in-house engine, used to publish several work-for-hire
 titles.
 
-![Screenshot of a 3D scene in the Godot Engine editor](https://raw.githubusercontent.com/godotengine/godot-design/master/screenshots/editor_tps_demo_1920x1080.jpg)
+Redot is a fork of Godot, intending to improve upon Godot in order to fulfill its potential and contribute to the shared codebase of both through a more genuinely community-driven model than Godot.
+<p align="center">
+	<img src="screenshot.jpg" width="900" alt="Redot Engine screenshot!">
+</p>
 
 ## Getting the engine
 
 ### Binary downloads
 
-Official binaries for the Godot editor and the export templates can be found
-[on the Godot website](https://godotengine.org/download).
+Official binaries for the Redot editor and the export templates will be found
+[on the Redot website](https://redotengine.org/download) once it's set up.
+
+Early builds are available [here](https://github.com/wesamdev/redot-dev-builds/releases), with no guarantee on their usability or stability.
```

Comment:
We do not have this, can you rtemove it. I don't think we will do this either to be honest.

Reply:
The redot website one?

Reply:
Done